packages/react-dom/src/__tests__/ReactDOMServerPartialHydration-test.internal.js

```diff
@@ -2563,4 +2564,49 @@ describe('ReactDOMServerPartialHydration', () => {
     // Now we're hydrated.
     expect(ref.current).not.toBe(null);
   });
+
+  it('should return fallback during server rendering when Portal has a suspense boundary ', () => {
+    const portalContainer = document.createElement('div');
```

Comment:
In a typical SSR setup, `document` won't be available, so `document.createElement` will throw. 

So it begs the question: what does a component that has a `createPortal` call looks like in an SSR setup? Because it won't look like these tests.

Reply:
Good point! Will have to rethink this. Thanks for the feedback.

Reply:
@sema can createPortal accept null if document.createElement doesn’t exist? The partial renderer won’t even need it. In the browser, we could throw if an element isn’t passed.

Reply:
It could but we currently don’t encourage any uses of “is server” conditionals since it’s easy to make mistakes that cause hydration issues as a result.

I was under the impression that we did something that rendered portals internally for SSR. But I think what we actually do is throw before we even get to it.

I think that’s probably the strategy here. Have something that throws before we even get to the code that needs to create a DOM node.

We just don’t have an API to do that.